app/containers/InAppNotification/IncomingCallNotification/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from 'react';
+import { Text, View, useWindowDimensions } from 'react-native';
+import Touchable from 'react-native-platform-touchable';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useDispatch } from 'react-redux';
+
+import { acceptCall, cancelCall } from '../../../actions/videoConf';
+import { ISubscription, SubscriptionType } from '../../../definitions';
+import i18n from '../../../i18n';
+import { getSubscriptionByRoomId } from '../../../lib/database/services/Subscription';
+import { useAppSelector } from '../../../lib/hooks';
+import { useEndpointData } from '../../../lib/hooks/useEndpointData';
+import { ESounds, useVideoConfRinger } from '../../../lib/hooks/useVideoConf';
+import { getRoomAvatar } from '../../../lib/methods/helpers';
+import { hideNotification } from '../../../lib/methods/helpers/notifications';
+import { CustomIcon } from '../../CustomIcon';
+import { CallHeader } from '../../VideoConf/CallHeader';
+import { useStyle } from './style';
+import { useTheme } from '../../../theme';
+
+export interface INotifierComponent {
+	notification: {
+		text: string;
+		payload: {
+			sender: { username: string };
+			type: SubscriptionType;
+		} & Pick<ISubscription, '_id' | 'name' | 'rid' | 'prid'>;
+		title: string;
+		avatar: string;
+	};
+	isMasterDetail: boolean;
+}
+
+const BUTTON_HIT_SLOP = { top: 12, right: 12, bottom: 12, left: 12 };
+
+const IncomingCallHeader = React.memo(
+	({ uid, callId, avatar, roomName }: { callId: string; avatar: string; uid: string; roomName: string }) => {
+		const [mic, setMic] = useState(true);
+		const [cam, setCam] = useState(false);
+		const dispatch = useDispatch();
+		const { playSound } = useVideoConfRinger(ESounds.DIALTONE);
+
+		const isMasterDetail = useAppSelector(state => state.app.isMasterDetail);
+		const styles = useStyle();
+
+		const insets = useSafeAreaInsets();
+		const { height, width } = useWindowDimensions();
+		const isLandscape = width > height;
+
+		useEffect(() => {
+			(() => playSound())();
```

Comment:
What happens if I'm listening to an audio or playing a video?

Reply:
The audio keeps playing, I tested it in another app and it also works like this.